treatment_auditor_cli.py

```
@@ -0,0 +1,922 @@
+    url = f"{backend}/cancer-research/v2/subtypes/{subtype_id}/modality-trials?{qs}"
+    try:
+        status, body = _http_request("GET", url, timeout=30)
+    except CLIError as e:
+        return modality, {"ok": False, "error": str(e), "trials": []}
+    if status != 200 or not body.get("success"):
+        return modality, {"ok": False, "error": body.get("error", f"HTTP {status}"), "trials": []}
+    return modality, {"ok": True, "trials": body.get("trials") or [], "condition": body.get("condition")}
+
+
+def _step_modality(backend: str, plan: Dict[str, Any], limit: int, progress: Progress) -> Dict[str, Any]:
+    progress.start("modality trials ×4")
```

Ruff (0.15.12)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)